coalahtml/_coalahtml/app/controllers/logs.js

```diff
@@ -9,5 +9,5 @@ angular.module('coalaHtmlApp')
         $scope.data = response.logs;
       });
     };
-    parseCoalaProject("tests/test_projects/simple");
+    parseCoalaProject("data");
```

Comment:
Also, probably, python part also knows about this folder, so it's a little duplication. You have raw constant in two files. If you'd want to change it, you'll have to change it in all places it's used. If you'll have it in one place you'll change it only once.